### DISCUSSION TITLE
Kennt sich jemand bei der config aus? (Wollte Teamliste)

### DISCUSSION THREAD

**Discussion by ErrorHunter1** [ScootKit/CustomDCBot#147] [Questions]
Ich wollte teamliste einstellen aber dort ist nur [] und bei der anderen ist nh haufen an zeilen die ich ned kapier wo wad gehört (versucht, bit sagt incorrect)

**Comment by hfgd123** [comment]
Sorry, but the official language of this repository is English. Please keep your discussions in English. Please read the readme file. If you still don't understand how to use the config, consider using the hosted version on scnx.xyz instead
